node/pkg/logscribeconsumer/app_test.go

```diff
@@ -111,70 +125,64 @@ func TestZeropglogWrite(t *testing.T) {
 }
 
 func TestBulkCopyLogEntries(t *testing.T) {
-	ctx := context.Background()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	go func() {
+		err := logscribe.Run(ctx)
+		if err != nil {
+			t.Errorf("Failed to start logscribe: %v", err)
+		}
+	}()
+
+	retrier.Retry(func() error {
+		resp, err := request.RequestRaw(request.WithEndpoint("http://localhost:3000/api/v1"))
+		if err != nil {
+			return err
+		}
+		if resp.StatusCode != http.StatusOK {
+			return fmt.Errorf("unexpected status code: %d", resp.StatusCode)
+		}
+		return nil
+	}, 10, 100*time.Millisecond, 1*time.Second)
```

Comment:
is there a specific reason retrier is required from the testcode?

Reply:
it takes a few hundred ms to initialize the fiber app, so the only options i could think of are (1) retry until the app is running or (2) time.sleep. Which one would you say is a better practice?

Reply:
let's time.sleep